src/pages/index.tsx

```diff
@@ -248,8 +293,10 @@ function Home() {
                 focusBorderColor="flame.600"
                 placeholder="Say something"
                 value={messageText}
+                maxLength={500}
                 onChange={(ev) => setMessageText(ev.target.value)}
               />
+              {messageText.length > 250 ? <small>{messageText.length}/500</small> : null}
```

Comment:
Ooo This is awesome. I'd like to see how this looks -- I've tweaked our Netlify settings to the next time you push it'll build a site preview for us to check out. In the future, could you please split out separate features like this into different PRs? Will make it faster to review/merge things piecemeal.

Reply:
One more thought: the 500 char limit is a pretty important global value. Could you please move it to https://github.com/wokenet/anchor/blob/main/constants.json?